### ISSUE TITLE
[2023 Yearly Social Media ] Call to action: Highlighting MicroProfilers at Events

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-marketing#769]
[Yearly Spreadsheet Media Inventory](https://docs.google.com/spreadsheets/d/1-j0SmFHSRw4pXMsdsY1x0R2HffIUZ3SQni_5zpoBdE8/edit#gid=1254202074)

**Media Description**

#MicroProfileSpeakers: Are you giving a #MicroProfile talk? If yes, your session needs social media (Twitter, Facebook & LinkedIn) global coverage!

Highlight your MicroProfile activities here: https://microprofile.io/resources/branding/

#OpenSource

**Schedule**
2 per month

**2022 issue**
https://github.com/eclipse/microprofile-marketing/issues/670

---
@rstjames, 

I new ticket will be created to improve the Resources >Branding page. Once that fix is done, the link of the above message brings the reader to the Website, under the Resources tab.

**Comment by aeiras** [comment]
@rstjames, 

Thank you for finding a good way to fix the branding page to use the direct link there of the website instead of the wiki. You rock!!!

**Comment by rstjames** [comment]
https://microprofile.io/resources/branding/#coverage
![2023_microprofile_session_coverage](https://user-images.githubusercontent.com/12132146/219704957-ac102901-63db-47f1-81c9-9db58b4b2cb3.jpg)

**Comment by rstjames** [comment]
Scheduled
